entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+echo "sleeping entrypoint"
+sleep 30
```

Comment:
This sleep should only happen when it’s localhost. No sleeps should be needed for production docker since the db is always on. 

This might be ok to merge now, but after merging if we notice the webapp makes user wait for 30 seconds a lot of times, this is why.

We only want sleeps happening in the docker-compose.yml. No sleeps should be needed to be triggered in the Dockerfile because they will then be triggered in the production container.

Reply:
Should only happen once on launch, but we can remember for the future if we have any issues.